src/app/pages/task-calendar/cloudsync/cloudsync-list/cloudsync-list.component.ts

```diff
@@ -135,6 +137,88 @@ export class CloudsyncListComponent implements InputTableConf {
           }
         });
       },
+    }, {
+      actionName: parentrow.description,
+      id: 'restore',
+      label: T('Restore'),
+      icon: 'restore',
+      onClick: (row) => {
+        console.log('restore');
+        const parent = this;
+        const conf: DialogFormConfiguration = {
+          title: T('Restore cloudsync task'),
```

Comment:
It could be confusing to have a public facing text string in this file when all the other public text strings are coming from the helptext files. 

```suggestion
          title: T('Restore Cloud Sync Task'),
```
Added title case for the dialog title and matched the spacing used for the Cloud Sync Tasks list screen.